addon/components/offering-form.js

```diff
@@ -439,28 +446,40 @@ export default Component.extend(ValidationErrorDisplay, Validations, {
 
     yield this.get('saveOffering').perform();
   }),
-  updateDurationMinutes: task(function * (minutes) {
-    let {validations} = yield this.validate();
-    this.send('addErrorDisplayFor', 'durationMinutes');
 
-    if (validations.get('durationMinutes.isInvalid')) {
+  updateDurationHours: task(function * (hours) {
+    yield timeout(DEBOUNCE_DELAY);
```

Comment:
What's the reason for a denouncing here? Since there isn't any async behavior I'm not sure why user input wouldn't be reflected immediately.

Reply:
I just checked. It's a non-issue as long as the validation check is working properly. When the validations were off, it was showing a value of 0 immediately after each entry so it was really confusing for users. So it was to remove the terrible UX.